pkg/awsclient/iam.go

```diff
@@ -130,15 +130,15 @@ func CheckIAMUserExists(reqLogger logr.Logger, client Client, userName string) (
 
 // CreateIAMUser creates a new IAM user in the target AWS account
 // Takes a logger, an AWS client for the target account, and the desired IAM username
-func CreateIAMUser(reqLogger logr.Logger, client Client, account *awsv1alpha1.Account, userName string) (*iam.CreateUserOutput, error) {
+func CreateIAMUser(reqLogger logr.Logger, client Client, account *awsv1alpha1.Account, userName string, managedTags []AWSTag) (*iam.CreateUserOutput, error) {
```

Comment:
Minor: Comment above should be updated to include new param `managedTags`

Reply:
Good catch!